modules/wordads/class-wordads.php

```diff
@@ -171,7 +196,9 @@ function init() {
 
 		if ( '/ads.txt' === $_SERVER['REQUEST_URI'] ) {
 
-			if ( false === ( $ads_txt_transient = get_transient( 'jetpack_ads_txt' ) ) ) {
+			$ads_txt_transient = get_transient( 'jetpack_ads_txt' );
```

Comment:
I guess this change is to make it a little clearer?

Reply:
Yup. WP Coding Standards don't allow setting a variable within a conditional, so breaking it out to make it clearer where they var is coming from.